pkg/internal/controller/controller.go

```diff
@@ -256,7 +261,7 @@ func (c *Controller) reconcileHandler(obj interface{}) bool {
 	c.Queue.Forget(obj)
 
 	// TODO(directxman12): What does 1 mean?  Do we want level constants?  Do we want levels at all?
-	c.Log.V(1).Info("Successfully Reconciled", "name", req.Name, "namespace", req.Namespace)
+	log.V(1).Info("Successfully Reconciled", "name", req.Name, "namespace", req.Namespace)
```

Comment:
Should we remove this as well, or increase verbosity to 4 or something?

Reply:
I think higher verbosity makes sense, I can see it coming in handy for debugging purposes, but generally it's just noise.